classes/core/EntityDAO.inc.php

```diff
@@ -113,22 +113,24 @@ public function fromRow(stdClass $row): DataObject
             }
         }
 
-        $rows = DB::table($this->settingsTable)
-            ->where($this->primaryKeyColumn, '=', $row->{$this->primaryKeyColumn})
-            ->get();
-
-        $rows->each(function ($row) use ($object, $schema) {
-            if (!empty($schema->properties->{$row->setting_name})) {
-                $object->setData(
-                    $row->setting_name,
-                    $this->convertFromDB(
-                        $row->setting_value,
-                        $schema->properties->{$row->setting_name}->type
-                    ),
-                    empty($row->locale) ? null : $row->locale
-                );
-            }
-        });
+        if ($this->settingsTable) {
```

Comment:
`jobs` doesnt have a `settingsTable`, so I've added this conditional here to avoid SQL errors.